config/initializers/sentry.rb

```diff
@@ -4,7 +4,8 @@
   SENTRY_PUBLIC_KEY = 'd35c8b2336084e67b08f7c4b5eeb54b9'
   SENTRY_PROJECT_ID = '1291517'
   config.dsn = "https://#{SENTRY_PUBLIC_KEY}@sentry.io/#{SENTRY_PROJECT_ID}"
-
+  # Send the environment to Sentry (development/testing/production)
+  config.environments = [Rails.env]
```

Comment:
I'm not sure if this will do example what we want. I believe that in the staging site, the `Rails.env` is still set to `production`.